modules/images/dominant-color/load.php

```diff
@@ -417,3 +417,1432 @@ function dominant_color_rgb_to_hex( $red, $green, $blue ) {
 
 	return sprintf( '%02x%02x%02x', $red, $green, $blue );
 }
+
+/**
+ * Integrates with the media library.
+ *
+ * @since n.e.x.t
+ */
+function dominant_color_add_media_library_integration() {
+	add_action( 'admin_footer', 'dominant_color_print_media_templates' );
+	add_filter( 'wp_prepare_attachment_for_js', 'dominant_color_prepare_attachment_for_js', 10, 3 );
+}
+add_filter( 'plugins_loaded', 'dominant_color_add_media_library_integration' );
+remove_action( 'admin_footer', 'wp_print_media_templates' );
+
+/**
+ * Add the dominant color when preparing attachment data.
+ *
+ * @since n.e.x.t
+ *
+ * @param array       $response   Array of prepared attachment data. @see wp_prepare_attachment_for_js().
+ * @param WP_Post     $attachment Attachment object.
+ * @param array|false $meta       Array of attachment meta data, or false if there is none.
+ * @return array Array of prepared attachment data.
+ */
+function dominant_color_prepare_attachment_for_js( $response, $attachment, $meta ) {
+	if ( ! wp_attachment_is_image( $attachment->ID ) ) {
+		return $response;
+	}
+
+	$dominant_color = dominant_color_get_dominant_color( $attachment->ID );
+	if ( ! $dominant_color ) {
+		return $response;
+	}
+
+	$response['dominantColor'] = $dominant_color;
+
+	// Set hasTransparency.
+	if ( dominant_color_has_transparency( $attachment->ID ) ) {
+		$response['hasTransparency']      = 'true';
+		$response['hasTransparencyClass'] = 'has-transparency';
+	} else {
+		$response['hasTransparency']      = 'false';
+		$response['hasTransparencyClass'] = 'no-transparency';
+	}
+
+	return $response;
+}
+
+/**
+ * Prints the templates used in the media manager.
+ *
+ * Copied from core's wp_print_media_templates() and modified to add the dominant color.
+ *
+ * Eslint is disabled because this block of code is copied from core and only included here to enable modification.
+ *
+ * @since n.e.x.t
+ */
+// phpcs:disable
+ function dominant_color_print_media_templates() {
```

Comment:
```suggestion
function dominant_color_print_media_templates() {
```

Remove space before the function name.